src/jit/liveness.cpp

```diff
@@ -2006,7 +2044,31 @@ void Compiler::fgComputeLifeLIR(VARSET_TP& life, BasicBlock* block, VARSET_VALAR
                     JITDUMP("Removing dead call:\n");
                     DISPNODE(call);
 
-                    node->VisitOperands([](GenTree* operand) -> GenTree::VisitResult {
+                    int       nodeCount     = 0;
+                    GenTree** disabledNodes = nullptr;
+
+                    if (call->fgArgInfo->HasStackArgs())
```

Comment:
Why do you need these iteration? Does it only determinate `disabledNodes` size? Why can't you use `stack` or `vector` instead?